vassal-doc/src/main/readme-referencemanual/ReferenceManual/Deck.adoc

```diff
@@ -84,19 +116,19 @@ May not be necessary or desirable if the Map Window contains a board on which th
 
 *Color:*  The color used to draw the rectangle representing the "tray" above.
 
-*Send Hotkey when empty?*  Select this option to send a Global Hotkey whenever the Deck is emptied.
+*Hotkey to send when Deck empties*  Select this option to send a Global Hotkey whenever the Deck is emptied.
```

Comment:
Isn't this now redundant with the next entry ("Hotkey to send when Deck empties:", line 121)?